.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: composer
+    directory: "/"
+    schedule:
+      interval: daily
```

Comment:
Given our infrequent release here I think weekly may be a better choice with less PRs to manage. I wonder if it's worth pulling in the [update-dependencies](https://github.com/ilios/ilios/blob/master/.github/workflows/update-dependencies.yaml) and maybe adding auto merge as well to keep the number even lower?

Reply:
changed to weekly and deps update workflow added :heavy_check_mark: